### PR TITLE
Updated web documentation (try 2)

### PR DESCRIPTION
**Pull request by ajhawkings**
This time with both commits! (@jlooper)

### CODE REVIEW

**Review by jlooper — COMMENTED** [review]
I'm not 100% sure this should be in this unit and not in the Browser unit content. @softchris what do you think?

File: 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.es.md [hunk 1]
```diff
@@ -171,8 +171,10 @@ Cuando un desarrollador quiere aprender algo nuevo, lo más probable es que recu
 
 #### Documentación popular sobre desarrollo de web
 
-- [Mozilla Developer Network](https://developer.mozilla.org/docs/Web)
```
Comment (Fronizzle) [inline 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.es.md:174]: https://developer.mozilla.org/en-US/docs/web

**Review by Fronizzle — APPROVED** [review]

**Review by friendyou1 — APPROVED** [review]
Chip or so D

**Comment by Chongruiyuan893** [comment]
> [CLA assistant check](https://cla.opensource.microsoft.com/microsoft/Web-Dev-For-Beginners?pullRequest=280) 
> All CLA requirements met.

File: 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.hi.md [hunk 2]
```diff
@@ -169,8 +169,10 @@ back add r0,r1
 
 #### वेब विकास पर लोकप्रिय प्रलेखन
 
-- [Mozilla Developer Network](https://developer.mozilla.org/docs/Web)
+- [Mozilla Developer Network (MDN)](https://developer.mozilla.org/docs/Web), Mozilla से, [Firefox](https://www.mozilla.org/firefox/) के प्रकाशक
 - [Frontend Masters](https://frontendmasters.com/learn/)
+- [Web.dev](https://web.dev), Google की ओर से, [Chrome](https://www.google.com/chrome/) के प्रकाशक
+- [Microsoft के अपने डेवलपर दस्तावेज़](https://docs.microsoft.com/microsoft-edge/#microsoft-edge-for-developers), के लिए [Microsoft Edge](https://www.microsoft.com/edge)
 
 ✅ कुछ शोध करें: अब जब आप एक वेब डेवलपर के वातावरण की मूल बातें जानते हैं, तो इसकी तुलना करें और वेब डिजाइनर के वातावरण के साथ इसके विपरीत करें।
```
Comment (iamsuryakant) [inline 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.hi.md:177]: Hindi Translation should be:

कुछ शोध करें: अब जब आप वेब डेवलपर के परिवेश की मूल बातें जानते हैं, तो इसकी तुलना वेब डिज़ाइनर के परिवेश से करें।

**Review by iamsuryakant — CHANGES_REQUESTED** [review]
Please update the Hindi Translation.

File: 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.es.md (hunk 1 above)
Comment (ajhawkings) [reply 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.es.md:174]: So no changes are needed?

Comment (iamsuryakant) [reply 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.es.md:174]: No

File: 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.hi.md (hunk 2 above)
Comment (ajhawkings) [reply 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.hi.md]: @iamsuryakant mark this as resolved if the new commit fixes this

Comment (iamsuryakant) [reply 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.hi.md]: Sure

**Review by iamsuryakant — APPROVED** [review]

**Review by Chongruiyuan893 — APPROVED** [review]

**Review by Chongruiyuan893 — APPROVED** [review]

**Review by jlooper — APPROVED** [review]
Sorry for the delay...had to really think about the items to include in this list

**Pull request merged**